disnake/utils.py

```diff
@@ -595,6 +595,28 @@ def compute_timedelta(dt: datetime.datetime):
     return max((dt - now).total_seconds(), 0)
 
 
+def iterable_is(iterable: Iterable[Any], tp: type) -> Union[Literal[True], Any]:
```

Comment:
Probably should be named `iterable_isinstance` (`is` is a different thingy) and maybe change `tp` annotation to `type | tuple[type | tuple[Any, ...], ...]`

Reply:
Added in 1b6f6ac8f24d484006d356af0c877055ff40573d (I used 3.10 annotations because the other one looks ugly)